obspy/io/reftek/core.py

```diff
@@ -237,7 +237,7 @@ def from_string(string):
             msg = "Ignoring incomplete packet."
             warnings.warn(msg)
             return None
-        bcd = codecs.encode(string[:16], "hex").decode("ASCII")
+        bcd = codecs.encode(string[:16], "hex_codec").decode("ASCII")
```

Comment:
uuups. Thanks!

Didn't even know that this was missing in py33. Astonishing, as it is part of py34 and py27.